pkg/volume/hostpath/host_path_test.go

```diff
@@ -154,9 +156,11 @@ func TestDeleterTempDir(t *testing.T) {
 }
 
 func TestProvisioner(t *testing.T) {
-	tempPath := fmt.Sprintf("/tmp/hostpath/%s", uuid.NewUUID())
+	tempPath, err := ioutil.TempDir("/tmp", "hostpath_test.")
```

Comment:
What is tempPath actually used for?  Is it just for finding the plugin?  Do we actually need the path to be valid?

Reply:
It seems that tempPath is just used for founding the plugin and not validated. Do you means we should validate it or need not make the tempPath?

Reply:
I mean we don't need to create the temp directory if we're not actually using it in the test.

Reply:
Done